unable to compile from source on RPi3

unable to compile from source on RPi3

`Log started at: Fri  7 Jul 13:07:46 UTC 2017

RetroPie-Setup version: 4.2.9 (70e6667)
System: Linux retropie 4.9.35-v7+ #1014 SMP Fri Jun 30 14:47:43 BST 2017 armv7l GNU/Linux
/home/pi/RetroPie-Setup/tmp/build/lr-beetle-ngp /home/pi

= = = = = = = = = = = = = = = = = = = = =
Getting sources for 'lr-beetle-ngp' : Neo Geo Pocket(Color)emu - Mednafen Neo Geo Pocket core port for libretro
= = = = = = = = = = = = = = = = = = = = =

git clone --recursive --depth 1 "https://github.com/libretro/beetle-ngp-libretro.git" "/home/pi/RetroPie-Setup/tmp/build/lr-beetle-ngp"
Cloning into '/home/pi/RetroPie-Setup/tmp/build/lr-beetle-ngp'...
/home/pi
/home/pi/RetroPie-Setup/tmp/build/lr-beetle-ngp /home/pi

= = = = = = = = = = = = = = = = = = = = =
Building 'lr-beetle-ngp' : Neo Geo Pocket(Color)emu - Mednafen Neo Geo Pocket core port for libretro
= = = = = = = = = = = = = = = = = = = = =

rm -f mednafen_ngp_libretro.so ./mednafen/ngp/biosHLE.o ./mednafen/ngp/dma.o ./mednafen/ngp/flash.o ./mednafen/ngp/gfx.o ./mednafen/ngp/system.o ./mednafen/ngp/interrupt.o ./mednafen/ngp/mem.o ./mednafen/ngp/rom.o ./mednafen/ngp/sound.o ./mednafen/ngp/T6W28_Apu.o ./mednafen/ngp/Z80_interface.o ./mednafen/sound/Blip_Buffer.o ./mednafen/state.o ./mednafen/mempatcher.o ./mednafen/sound/Stereo_Buffer.o ./libretro.o ./mednafen/ngp/bios.o ./mednafen/ngp/TLCS-900h/TLCS900h_disassemble.o ./mednafen/ngp/TLCS-900h/TLCS900h_disassemble_extra.o ./mednafen/ngp/TLCS-900h/TLCS900h_disassemble_reg.o ./mednafen/ngp/TLCS-900h/TLCS900h_disassemble_dst.o ./mednafen/ngp/TLCS-900h/TLCS900h_disassemble_src.o ./mednafen/ngp/TLCS-900h/TLCS900h_interpret.o ./mednafen/ngp/TLCS-900h/TLCS900h_interpret_dst.o ./mednafen/ngp/TLCS-900h/TLCS900h_interpret_reg.o ./mednafen/ngp/TLCS-900h/TLCS900h_interpret_single.o ./mednafen/ngp/TLCS-900h/TLCS900h_interpret_src.o ./mednafen/ngp/TLCS-900h/TLCS900h_registers.o ./mednafen/hw_cpu/z80-fuse/z80_ops.o ./mednafen/hw_cpu/z80-fuse/z80.o ./mednafen/ngp/rtc.o ./mednafen/file.o ./mednafen/settings.o
g++ -c -omednafen/ngp/biosHLE.o mednafen/ngp/biosHLE.cpp  -O2 -march=armv8-a+crc -mtune=cortex-a53 -mfpu=neon-fp-armv8 -mfloat-abi=hard -ftree-vectorize -funsafe-math-optimizations -U__GCC_HAVE_SYNC_COMPARE_AND_SWAP_2 -DGIT_VERSION=\"" 74d6e3d"\" -DWANT_16BPP -DWANT_STEREO_SOUND -DFRONTEND_SUPPORTS_RGB565 -O2 -DNDEBUG -funroll-loops -fPIC  -I. -I./mednafen -I./mednafen/include -I./mednafen/intl -I./mednafen/hw_sound -I./mednafen/hw_cpu -I./mednafen/hw_misc -I./libretro-common/include   -DSIZEOF_DOUBLE=8 -Wall -Wno-sign-compare -Wno-unused-variable -Wno-unused-function -Wno-uninitialized  -Wno-strict-aliasing -DMEDNAFEN_VERSION=\"0.9.31\" -DPACKAGE=\"mednafen\" -DMEDNAFEN_VERSION_NUMERIC=931 -DPSS_STYLE=1 -DMPC_FIXED_POINT -DSTDC_HEADERS -D__STDC_LIMIT_MACROS -D__LIBRETRO__ -D_LOW_ACCURACY_  -DINLINE="inline"
g++ -c -omednafen/ngp/dma.o mednafen/ngp/dma.cpp  -O2 -march=armv8-a+crc -mtune=cortex-a53 -mfpu=neon-fp-armv8 -mfloat-abi=hard -ftree-vectorize -funsafe-math-optimizations -U__GCC_HAVE_SYNC_COMPARE_AND_SWAP_2 -DGIT_VERSION=\"" 74d6e3d"\" -DWANT_16BPP -DWANT_STEREO_SOUND -DFRONTEND_SUPPORTS_RGB565 -O2 -DNDEBUG -funroll-loops -fPIC  -I. -I./mednafen -I./mednafen/include -I./mednafen/intl -I./mednafen/hw_sound -I./mednafen/hw_cpu -I./mednafen/hw_misc -I./libretro-common/include   -DSIZEOF_DOUBLE=8 -Wall -Wno-sign-compare -Wno-unused-variable -Wno-unused-function -Wno-uninitialized  -Wno-strict-aliasing -DMEDNAFEN_VERSION=\"0.9.31\" -DPACKAGE=\"mednafen\" -DMEDNAFEN_VERSION_NUMERIC=931 -DPSS_STYLE=1 -DMPC_FIXED_POINT -DSTDC_HEADERS -D__STDC_LIMIT_MACROS -D__LIBRETRO__ -D_LOW_ACCURACY_  -DINLINE="inline"
g++ -c -omednafen/ngp/flash.o mednafen/ngp/flash.cpp  -O2 -march=armv8-a+crc -mtune=cortex-a53 -mfpu=neon-fp-armv8 -mfloat-abi=hard -ftree-vectorize -funsafe-math-optimizations -U__GCC_HAVE_SYNC_COMPARE_AND_SWAP_2 -DGIT_VERSION=\"" 74d6e3d"\" -DWANT_16BPP -DWANT_STEREO_SOUND -DFRONTEND_SUPPORTS_RGB565 -O2 -DNDEBUG -funroll-loops -fPIC  -I. -I./mednafen -I./mednafen/include -I./mednafen/intl -I./mednafen/hw_sound -I./mednafen/hw_cpu -I./mednafen/hw_misc -I./libretro-common/include   -DSIZEOF_DOUBLE=8 -Wall -Wno-sign-compare -Wno-unused-variable -Wno-unused-function -Wno-uninitialized  -Wno-strict-aliasing -DMEDNAFEN_VERSION=\"0.9.31\" -DPACKAGE=\"mednafen\" -DMEDNAFEN_VERSION_NUMERIC=931 -DPSS_STYLE=1 -DMPC_FIXED_POINT -DSTDC_HEADERS -D__STDC_LIMIT_MACROS -D__LIBRETRO__ -D_LOW_ACCURACY_  -DINLINE="inline"
g++ -c -omednafen/ngp/gfx.o mednafen/ngp/gfx.cpp  -O2 -march=armv8-a+crc -mtune=cortex-a53 -mfpu=neon-fp-armv8 -mfloat-abi=hard -ftree-vectorize -funsafe-math-optimizations -U__GCC_HAVE_SYNC_COMPARE_AND_SWAP_2 -DGIT_VERSION=\"" 74d6e3d"\" -DWANT_16BPP -DWANT_STEREO_SOUND -DFRONTEND_SUPPORTS_RGB565 -O2 -DNDEBUG -funroll-loops -fPIC  -I. -I./mednafen -I./mednafen/include -I./mednafen/intl -I./mednafen/hw_sound -I./mednafen/hw_cpu -I./mednafen/hw_misc -I./libretro-common/include   -DSIZEOF_DOUBLE=8 -Wall -Wno-sign-compare -Wno-unused-variable -Wno-unused-function -Wno-uninitialized  -Wno-strict-aliasing -DMEDNAFEN_VERSION=\"0.9.31\" -DPACKAGE=\"mednafen\" -DMEDNAFEN_VERSION_NUMERIC=931 -DPSS_STYLE=1 -DMPC_FIXED_POINT -DSTDC_HEADERS -D__STDC_LIMIT_MACROS -D__LIBRETRO__ -D_LOW_ACCURACY_  -DINLINE="inline"
g++ -c -omednafen/ngp/system.o mednafen/ngp/system.cpp  -O2 -march=armv8-a+crc -mtune=cortex-a53 -mfpu=neon-fp-armv8 -mfloat-abi=hard -ftree-vectorize -funsafe-math-optimizations -U__GCC_HAVE_SYNC_COMPARE_AND_SWAP_2 -DGIT_VERSION=\"" 74d6e3d"\" -DWANT_16BPP -DWANT_STEREO_SOUND -DFRONTEND_SUPPORTS_RGB565 -O2 -DNDEBUG -funroll-loops -fPIC  -I. -I./mednafen -I./mednafen/include -I./mednafen/intl -I./mednafen/hw_sound -I./mednafen/hw_cpu -I./mednafen/hw_misc -I./libretro-common/include   -DSIZEOF_DOUBLE=8 -Wall -Wno-sign-compare -Wno-unused-variable -Wno-unused-function -Wno-uninitialized  -Wno-strict-aliasing -DMEDNAFEN_VERSION=\"0.9.31\" -DPACKAGE=\"mednafen\" -DMEDNAFEN_VERSION_NUMERIC=931 -DPSS_STYLE=1 -DMPC_FIXED_POINT -DSTDC_HEADERS -D__STDC_LIMIT_MACROS -D__LIBRETRO__ -D_LOW_ACCURACY_  -DINLINE="inline"
mednafen/ngp/system.cpp:3:20: fatal error: Stdlib.h: No such file or directory
 #include <Stdlib.h>
                    ^
compilation terminated.
Makefile:352: recipe for target 'mednafen/ngp/system.o' failed
make: *** [mednafen/ngp/system.o] Error 1
make: *** Waiting for unfinished jobs....
/home/pi
Could not successfully build lr-beetle-ngp - Neo Geo Pocket(Color)emu - Mednafen Neo Geo Pocket core port for libretro (/home/pi/RetroPie-Setup/tmp/build/lr-beetle-ngp/mednafen_ngp_libretro.so not found).

Log ended at: Fri  7 Jul 13:07:53 UTC 2017
Total running time: 0 hours, 0 mins, 7 secs
`

--- a/mednafen/ngp/system.cpp
+++ b/mednafen/ngp/system.cpp
@@ -1,6 +1,6 @@
 #include <stdio.h>
 #include <stdint.h>
-#include <Stdlib.h>
+#include <stdlib.h>
 #include <string.h>
 #include "flash.h"
 #include "system.h"

